### ISSUE TITLE
 running xatlas to unwrap UVs for mesh for a very long time

### ISSUE THREAD

**Issue by SilkySnark** [ashawkey/stable-dreamfusion#27]
Hey all,
Awesome project thanks for sharing.

I am having a super long run in the xatlas unwrap - around an hour and still running.
Is this expected? anything I can do to reduce this time?
Thanks!
![image](https://user-images.githubusercontent.com/93453066/194774905-8c374ffd-4740-4d7c-a00f-f07b9e3fb8fb.png)

**Comment by ashawkey** [comment]
@SilkySnark Yes, it takes very long time if the mesh is too large. For me it seems the network is not well converged, a mesh having 6324306 faces is not normal.
